utils/common_utils.py

```diff
@@ -61,3 +61,9 @@ def run_command(command, working_dir):
         cmd = subprocess.Popen(command, cwd=working_dir, stdout=null_out, stderr=subprocess.STDOUT,
                                stdin=subprocess.PIPE)
     cmd.wait()
+
+def get_year() -> str:
+    """
+    Get the current year. 
+    """
+    return str(datetime.datetime.now().year)
```

Comment:
**Nitpick**: Could we add the return parameter to the docstring either as is or place the return statement in a variable and add the return parameter to the docstring?

Maybe something like this:
```python
def get_year() -> str:
    """
    Get the current year.
 
    Returns:
        current_year The current year formatted as a string
    """
    current_year = str(datetime.datetime.now().year)
    return current_year
```
Or
```python
def get_year() -> str:
    """
    Get the current year.
 
    Returns:
         str:  The current year formatted as a string
    """
    return str(datetime.datetime.now().year)
```